src/components/ColorPicker.jsx

```diff
@@ -41,7 +41,6 @@ const ColorPicker = (props) => {
       >
         <Button
           size='small'
-          shape='circle'
```

Comment:
does removing this still keep the shape circular?

Reply:
Better make them circles, Vicky didn't change them to squares, I wouldn't be making design changes unless it is seen as an issue by users

Reply:
Sure.